db/migrations/0002_user_order_ticket_alter_movie_actors_and_more.py

```diff
@@ -0,0 +1,114 @@
+# Generated by Django 4.0.2 on 2024-12-25 17:43
+
+from django.conf import settings
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('password', models.CharField(max_length=128, verbose_name='password')),
```

Comment:
Avoid setting passwords directly on the `User` object. Use the `set_password` method to ensure passwords are hashed securely.